tests/e2e/utils/data_fetcher/__init__.py

```diff
@@ -0,0 +1,6 @@
+from tests.e2e.utils.data_fetcher.common import get_lambda_response
+from tests.e2e.utils.data_fetcher.logs import get_logs
+from tests.e2e.utils.data_fetcher.metrics import get_metrics
+from tests.e2e.utils.data_fetcher.traces import get_traces
+
+__all__ = ["get_traces", "get_metrics", "get_logs", "get_lambda_response"]
```

Comment:
since you do an explicit method import in lines 1-4 do we need __all__ list? Either we can remove it or simplify imports by doing `from tests.e2e.utils.data_fetcher import *`

Reply:
I know I don't like either. Since you too are bothered, lemme play with flake8 to ignore unused imports in `__init__.py` files.

Reply:
Solved it with a new feature in flake8 - per-file-ignore ;)